### PR TITLE
rawtherapee: 5.0-r1 -> 5.2

### PR DESCRIPTION
**Pull request by markus1189**
###### Motivation for this change

Update to the latest version of rawtherapee

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Also added some flags found in http://rawpedia.rawtherapee.com/Linux#Compile_RawTherapee

### CODE REVIEW

File: pkgs/applications/graphics/rawtherapee/default.nix [hunk 1]
```diff
@@ -22,6 +22,8 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DPROC_TARGET_NUMBER=2"
+    "-DCACHE_NAME_SUFFIX=\"\""
+    "-DCMAKE_BUILD_TYPE=\"release\""
```
Comment (Mic92) [inline pkgs/applications/graphics/rawtherapee/default.nix]: `release` is the default build type: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L52

Comment (markus1189) [reply pkgs/applications/graphics/rawtherapee/default.nix]: Will fix it this evening, thanks!

**Comment by markus1189** [comment]
Fixed!

**Comment by rycee** [comment]
Works well for me. Rebased into master in 572361f3960758d77bc69c810c797e842d4af51b. Thanks for the contribution!

**Pull request closed without merging**